### DISCUSSION TITLE
about use telnet

### DISCUSSION THREAD

**Discussion by gumozhihu** [Eugeny/tabby#6248] [Release Notes]
When Telnet is used to connect to Linux, the entered commands are duplicated and cannot be completed using the TAB key
![tabby](https://user-images.githubusercontent.com/104196063/164674079-b5d9e3ee-e800-4484-821a-f41b43e78be8.png)

**Comment by gumozhihu** [comment]
when you input one string，it becomes two.and You cannot use the trial tab key to complete the content

**Comment by martin-calvin** [comment]
i have same problem
